articles-2.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>

```

Comment:
Duplicated Page 1. I haven't worked on it much but I changed the bottom navigation button links.